website/docs/07-apps/01-included-in-onion/pico8-wrapper.md

```diff
@@ -0,0 +1,115 @@
+---
+slug: /apps/pico8-wrapper
+description: PICO-8 Native Wrapper by XK9274
+---
+
+:::caution 

```

Comment:
Hi, thanks again for putting this info together, it's great. Only a few little things

Can you remove this caution block please? we shouldn't need it now.